### ISSUE TITLE
[Bug]: MissingMethodException

### ISSUE THREAD

**Issue by tomasfil** [reactiveui/ReactiveUI.Validation#696]
### Describe the bug 🐞

There is some kind if version missmatch for ReactiveUI and ReactiveUI.Validation.

Most likely DynamicData when adding ReactiveUI over Validation:
![image](https://github.com/user-attachments/assets/db3ed011-0e20-44a9-afda-a922349a83e6)

```
Method not found: 'System.Collections.Generic.IEnumerable`1<!0> DynamicData.IObservableList`1.get_Items()'.
 at ReactiveUI.Validation.Contexts.ValidationContext.BuildText() in D:\a\ReactiveUI.Validation\ReactiveUI.Validation\src\ReactiveUI.Validation\Contexts\ValidationContext.cs:line 233
   at ReactiveUI.Validation.Contexts.ValidationContext.<.ctor>b__8_3(Boolean _) in D:\a\ReactiveUI.Validation\ReactiveUI.Validation\src\ReactiveUI.Validation\Contexts\ValidationContext.cs:line 71
   at System.Reactive.Linq.ObservableImpl.Select`2.Selector._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Select.cs:line 39
```

### Step to reproduce

Run the attached minimal repro repo.

### Reproduction repository

https://github.com/tomasfil/ReactiveUI.Validation.MissingMethod

### Expected behavior

This shouldnt happen...

**Comment by tomasfil** [comment]
I think it might be fixed with?
https://github.com/reactiveui/ReactiveUI.Validation/pull/692

**Comment by Nextra** [comment]
Fix is here: #691

Just needs a release now.

**Comment by ChrisPulman** [comment]
Please update to version 4.1.1 which should resolve this
